features/update/activities/feature.ts

```diff
@@ -0,0 +1,38 @@
+import { Feature } from '@temporalio/harness';
+import * as wf from '@temporalio/workflow';
+import * as assert from 'assert';
+
+const myUpdate = wf.defineUpdate<void, any[]>('myUpdate');
+const activityCount = 5;
+const activityResult = 6;
+
+const activitiesImpl = {
+  async myActivity(): Promise<number> {
+    return activityResult;
+  },
+};
+
+const activities = wf.proxyActivities<typeof activitiesImpl>({
+  startToCloseTimeout: '5s',
+});
+
+export const feature = new Feature({
+  workflow,
+  activities: activitiesImpl,
+  checkResult: async (_, handle) => {
+    await handle.executeUpdate(myUpdate, {});
```

Comment:
We should not require an options object if don't need to set any fields on one

Reply:
Good catch, fixed!